docs/user-guide.md

```diff
@@ -5,8 +5,17 @@ This is the user guide for Open-Local.
 ## Requirements
 
 - Kubernetes v1.18+
-- helm v3.0+
+- Helm v3.0+
 - [lvm2](https://en.wikipedia.org/wiki/Logical_Volume_Manager_(Linux))
+- At least one block device
+
+## Configuring
+
+You'd better provide one block device for **each node**, which will be used for creating VolumeGroup by Open-Local.
```

Comment:
Avoid using second-person pronouns in written language, eg
> to create a Volume Group by Open Local automatically, edit xxxx

Reply:
done